core/hybrisadaptor.h

```diff
@@ -204,11 +204,7 @@ class HybrisManager : public QObject
 #else
     // HAL backend
     struct sensors_module_t      *m_halModule;
-#ifdef SENSORS_DEVICE_API_VERSION_1_0
     sensors_poll_device_1_t      *m_halDevice;
-#else
-    struct sensors_poll_device_t *m_halDevice;
```

Comment:
This looks like: drop support for devices where SENSORS_DEVICE_API_VERSION_1_0 is not defined (and would sort of explain the identical if/else paths). Is that the intent / are there still devices like that?

Reply:
SENSORS_DEVICE_API_VERSION_1_0 was added in Android 4.3 and we don't support older Android bases anymore.